docs/reference.rst

```diff
@@ -692,6 +692,8 @@ Faker
         >>> user.name
         'Lucy Cechtelar'
 
+    Visit the `faker documentation <https://faker.readthedocs.io/en/latest//>`_
+    for a full reference of Faker attributes.
```

Comment:
What do you think about replacing the link to PyPI located line 678 with https://faker.readthedocs.io/en/latest/ instead?
Since Faker is necessarily installed with factory boy, users will be more interested by faker's documentation than installation instruction and a basic readme on PyPI.

Reply:
Agree with ^^. I don't mind this PR at all, but it probably does make sense to just point them straight to the docs from the getgo.